build.gradle

```diff
@@ -74,6 +75,8 @@ dependencies {
   compile "org.slf4j:slf4j-jdk14:1.7.25"
   compile "org.json:json:20190722"
   compile "com.googlecode.java-diff-utils:diffutils:1.3.0"
+  compile "org.apache.commons:commons-text:1.9"
```

Comment:
It looks like this addition also updated the version of commons-lang3. Can you double check test: `StackTraceEnricherTest`.

I suspect we should update the listed version in `  compile "org.apache.commons:commons-lang3:3.9"` to 3.11

And change the test to expect 3.11 instead of 3.9

Reply:
Will have a look at it and get back to you.

Reply:
I have checked commons-text:1.9 and commons-lang3:3.11 for known vulnerabilities against https://ossindex.sonatype.org/ and updated dependency + test.